<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devMateUI</class>
 <widget class="QWidget" name="devMateUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dev' Mate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="tkDMFunctionsLE">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Presets...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>------------------</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*rig*,*create*,*character*</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tkDMFiltersTB">
       <property name="text">
        <string>*rig*,*create*</string>
       </property>
       <property name="+cc" stdset="0">
        <string>&quot;tkDevHelpers.filtersChanged()&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="tkDMDocCB">
       <property name="text">
        <string>Search in docs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="+cc" stdset="0">
        <string>&quot;tkDevHelpers.docChanged()&quot;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="tkDMFunctionsLB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="+sc" stdset="0">
        <string>&quot;tkDevHelpers.functionSelChanged()&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tkDMFunctionsTB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>pictures/MiniLogo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tkDMFunctionsLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tkDMFunctionsTB</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
